pkgs/top-level/all-packages.nix

```diff
@@ -5861,6 +5861,8 @@ let
 
   folly = callPackage ../development/libraries/folly { };
 
+  fomp = callPackage ../applications/audio/fomp { };
```

Comment:
I guess this belongs in the applications section rather than the libraries section, right?